shim.h

```diff
@@ -81,5 +83,9 @@ static inline CURLcode curlHelperGetInfoLong(CURL *curl, CURLINFO info, long *da
     return curl_easy_getinfo(curl, info, data);
 }
 
+static inline CURLcode curlHelperSetMultiOpt(CURLM *curlMulti, CURLMoption option, long data) {
```

Comment:
Should this return `CURLMcode` to match the signature of `curl_multi_setopt()`?  See https://curl.haxx.se/libcurl/c/curl_multi_setopt.html

Reply:
Good point. Fixed the return type.